config/default_values.toml

```diff
@@ -15,7 +15,7 @@ host_and_port = "0.0.0.0:7600"
 url_prefix = "/"
 
 [service.tap]
-max_receipt_value = 1000000000000000 # 0.001 GRT
+max_receipt_value_grt = 1000000000000000 # 0.001 GRT
```

Comment:
Can we use the custom GRT to u128 deserializer here?

Reply:
Wow, thanks for showing that. Actually we are converting to GRT this is `1000000000000000 GRT`. Let me fix that

Reply:
Oof that'd be a lot of GRT :grimacing: